canonical-websites/microk8s.io
# Microk8s homepage refresh 19

## Done

Created new index page with Microk8s from  [the copy doc](https://docs.google.com/document/d/1ObWmhRc4GCIhrBi8gT7yIIs0J_dn_8ravfLtcwhr3uQ/edit#) and [design spec](https://github.com/ubuntudesign/web-squad/issues/915)
## QA

- Check out this feature branch
- Run the site using the command `./run`
- View the site locally in your web browser at: [http://0.0.0.0:8027/](http://0.0.0.0:8027/)
- Run through the following [QA steps](https://github.com/canonical-webteam/practices/blob/master/workflow/qa-steps.md)
- Make sure the MicroK8s matches the  [the copy doc](https://docs.google.com/document/d/1ObWmhRc4GCIhrBi8gT7yIIs0J_dn_8ravfLtcwhr3uQ/edit#) and [design spec](https://github.com/ubuntudesign/web-squad/issues/915


## Issue / Card

Fixes https://github.com/canonical-websites/microk8s.io/issues/25
## Screenshots

![image](https://user-images.githubusercontent.com/43535482/48898821-aef13e80-ee45-11e8-91b1-35198ead6d42.png)

![image](https://user-images.githubusercontent.com/43535482/48898847-c4666880-ee45-11e8-9a16-82085ebcb7bf.png)

![image](https://user-images.githubusercontent.com/43535482/48898862-d6e0a200-ee45-11e8-8298-f3d09c5d56eb.png)
